static/js/admin-versions.js

```diff
@@ -0,0 +1 @@
+import './admin/l10n.js';
```

Comment:
similar question to the css files - why do we need to have this layer of misdirection for a single file?

Reply:
Addressed in the original comment.